js/forum/src/components/EditUserModal.js

```diff
@@ -33,26 +34,55 @@ export default class EditUserModal extends Modal {
   }
 
   content() {
+    const topObj = this;
+    const activateButton = Button.component({
+      className: 'Button Button--block',
+      children: app.translator.trans('core.forum.edit_user.activate'),
+      loading: topObj.loading,
+      onclick: function() {
+        topObj.loading = true;
+        const data = {
+          username: topObj.username(),
+          isActivated: true,
+        };
+        topObj.props.user.save(data, {errorHandler: topObj.onerror.bind(topObj)})
+          .then(() => {
+            topObj.isActivated(true);
+            topObj.loading = false;
+            m.redraw();
+          })
+          .catch(() => {
```

Comment:
I am just copy-pasting from `onSubmit()`. So I guess that part (`onSubmit()`) should also be changed?